bin/get-scitokens-mapfile

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Create a scitokens issuer -> unix user mapfile from topology data
+"""
+
+import os
+import sys
+
+if __name__ == "__main__" and __package__ is None:
+    _parent = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    sys.path.append(_parent + "/src")
+
+from webapp.common import is_null
+import webapp.models
+
+from argparse import ArgumentParser
+
+
+def main(argv):
+    parser = ArgumentParser(description=__doc__)
+    parser.add_argument(
+        "--outfile",
+        help="File to write; uses '-' (i.e. stdout) by default.",
+        default="-",
+    )
+    parser.add_argument(
+        "--strict",
+        action="store_true",
+        help="Bail on invalid scitoken or on no scitokens info. "
+        "Otherwise just add a comment to the result.",
+    )
+
+    args = parser.parse_args(argv[1:])
+
+    mapfile = ""
+    all_vos_data = webapp.models.GlobalData().get_vos_data()
+    for vo_name, vo_data in all_vos_data.vos.items():
+        if is_null(vo_data, "Credentials", "TokenIssuers"):
+            continue
+        mapfile += f"# {vo_name}\n"
+        for token_issuer in vo_data["Credentials"]["TokenIssuers"]:
+            url = token_issuer.get("URL")
+            unix_user = token_issuer.get("DefaultUnixUser")
+            if url and unix_user:
+                mapfile += f"SCITOKEN {url} {unix_user}\n"
+            else:
+                mapfile += f"# invalid SCITOKEN: {url or '<NO URL>'} {unix_user or '<NO UNIX USER>'}\n"
```

Comment:
Cool, I didn't know you could do that with fstrings. Do these comments get printed for VOs without `Credentials.TokenIssuers`?

Reply:
Nope, the `is_null()` above takes care of skipping those. And yeah, fstrings are great.